modules/connectivity/locals.tf

```diff
@@ -839,6 +840,18 @@ locals {
     location =>
     "${local.azfw_pip_resource_id_prefix[location]}/${local.azfw_pip_name[location]}"
   }
+  azfw_pip_zones = {
+    for location in local.hub_network_locations :
+    location =>
+    try(
+      local.custom_settings.azurerm_public_ip["connectivity_firewall"][location].zones,
```

Comment:
is this needed? 

My thinking is we would always want to either convert the user input to 1 or 3 zones due to the api behaviour. So therefore do we want to let a customer still override this with advanced?

As my understanding that if they provide this value in advanced it would be used as try fins the first no error value?